jp2_pc/Source/Lib/GeomDBase/PartitionPriv.hpp

```diff
@@ -289,10 +289,7 @@ struct SPartitionSettings
 
 private:
 
-	class CPartition::CPriv;
-
 	friend class CPartition;
-	friend class CPartition::CPriv;
 	friend float CPartitionSpace::fDistanceFromGlobalCameraSqr() const;
 	friend float CInstance::fDistanceFromGlobalCameraSqr() const;
 
```

Comment:
`CPriv` is declared as private in `CPartition` and not accessible here. This entire friend declaration seems to be unused anyway, so it is removed.